### DISCUSSION TITLE
Looking for peer review of this mathematically provable unbreakable cryptography concept. (Seems obvious right?)

### DISCUSSION THREAD

**Discussion by HomingHamster** [pkic/community#33] [Post-Quantum Cryptography]
https://circuspam.coffee/2024/08/18/a-prototype-for-quantum-secure-mathematically-uncrackable-encryption/

**Comment by primetomas** [comment]
Since all symmetric key encryption schemes, such as AES, are quantum safe. Quantum safeness does not seem as a very unique feature for the paper? I think it need to describe why you would use this instead of say AES?

**Comment by HomingHamster** [comment]
Thanks for your attention, I didn't know AES was provably unbreakable? Also this paper describes digital attack vectors possible on any encryption possible within the universe (I.E. no infinite random). It’s intended as an unbreakable scheme not a replacement for HTTPS.
